gruntfile.js

```diff
@@ -13,7 +13,7 @@ module.exports = function(grunt) {
       options: {
         jshintrc: true
       },
-      all: ['src/app/**/*.js']
+      all: ['src/app/**/main.js']
```

Comment:
why only linting main.js? this should be restored to run jshint on all .js files.